std_msgs/Header header

float32 identifier

float32 ri

float32 angleSFE

geometry_msgs/Pose pose

